features/facebookImport/constants/constants.js

```diff
@@ -0,0 +1,3 @@
+export const INITIAL_HISTORY_STATE = {
```

Comment:
Why not simply a file constants.js in src?

Reply:
Considering there's just a single constant, I wouldn't even give it its own file, but that's a nit, I can live with it :D

Reply:
There's two now with the namespace :D